lib/livebook_web/live/file_select_component.ex

```diff
@@ -267,13 +270,14 @@ defmodule LivebookWeb.FileSelectComponent do
       <:toggle>
         <button class="w-full flex space-x-2 items-center p-2 rounded-lg hover:bg-gray-100 focus:ring-1 focus:ring-gray-400"
           data-toggle
+          aria-label={"#{if @file_info.name == "..", do: "parent directory", else: @file_info.name}"}
           phx-click="set_path"
           phx-value-path={@file_info.file.path}
           phx-target={@myself}>
           <span class="block">
             <.remix_icon icon={@icon} class={"text-xl align-middle #{if(@file_info.is_running, do: "text-green-300", else: "text-gray-400")}"} />
           </span>
-          <span class={"flex font-medium overflow-hidden whitespace-nowrap #{if(@file_info.is_running, do: "text-green-300", else: "text-gray-500")}"}>
+          <span id={"file-info-path-#{@file_info.name}"} class={"flex font-medium overflow-hidden whitespace-nowrap #{if(@file_info.is_running, do: "text-green-300", else: "text-gray-500")}"}>
```

Comment:
Do we need this id?

Reply:
What do you mean by it was already there? :D

Reply:
This id was already in the code but without the file info part.

Reply:
I'm confused, because I can't see it in the diff :see_no_evil: Either way if we don't need the id I'd remove it :)

Reply:
Maybe I’m crazy? Anyway, we don't need that and I will remove it.